sunpy/visualization/imageanimator.py

```diff
@@ -1006,21 +1006,26 @@ def _set_unit_in_axis(self, axes):
         if self.unit_x_axis is not None:
             axes.coords[2].set_format_unit(self.unit_x_axis)
             axes.coords[2].set_ticks(spacing = 10*u.arcmin, exclude_overlapping=True)
+            axes.coords[2].set_ticklabel_visible(True)
+            axes.coords[2].set_ticks_visible(True)
+
         if self.unit_y_axis is not None:
             axes.coords[1].set_format_unit(self.unit_y_axis)
             axes.coords[1].set_ticks(spacing = 10*u.arcmin, exclude_overlapping=True)
```

Comment:
I think spacing may fail if we have dimensionless, since we have to pass a `astropy.quantity` here